packages/create-gatsby/src/tracking.ts

```diff
@@ -28,7 +33,34 @@ export interface ITrackCliArgs {
 
 const sessionId = uuidv4()
 
+// Adapted from gatsby-telemetry
+export function isTrackingEnabled(): boolean {
+  // Cache the result
+  if (trackingEnabled !== undefined) {
+    return trackingEnabled
+  }
+
+  let enabled = store.get(`telemetry.enabled`) as boolean | null
+
+  if (enabled === undefined || enabled === null) {
+    enabled = true
+    store.set(`telemetry.enabled`, enabled)
+  }
+  trackingEnabled = enabled
+
+  if (trackingDisabledFromEnvVar) {
+    enabled = false
+    trackingEnabled = enabled
+  }
```

Comment:
```suggestion
  if (trackingDisabledFromEnvVar) {
    enabled = false
  }
  
  trackingEnabled = enabled
```

We can simplify it to this